portfolio/src/main/webapp/script.js

```diff
@@ -92,3 +94,21 @@ const handleDeleteButton = function() {
         deleteAllComments();
     });
 };
+
+const fetchBlobstoreUrlAndShowForm = function() {
+    fetch("/blobstore-upload-url").then((response) => {
+        return response.text();
+    }).then((imageUploadUrl) => {
+        const photoForm = document.getElementById('photo-upload-form');
+        photoForm.action = imageUploadUrl;
+        photoForm.classList.remove('hidden');
+    });
+};
+
+const fetchPhotoFormHandler = function() {
+    fetch("/photo-form-handler").then(response => response.json()).then((posts) => {
+        for(let post of posts) {
+            $('#photo-container').append('<div class="photo ' + post.display.toLowerCase()   + '"><p>' + post.message + '</p><img src="' + post.url + '"/></div>');
```

Comment:
Nit: I think you have a couple of extra spaces after `post.display.toLowerCase()` and before the `+`

Also, this line is a bit long, I'd suggest breaking it up into lines. So something like:
```
$('#photo-container').append(
    '<div class="photo ' + post.display.toLowerCase() + '"><p>' +
    post.message + '</p><img src="' + post.url + '"/></div>');
```

Or whatever would make sense to you that would keep the line from being over an 80-column limit.